generators/ruby/codegen/src/ast/ExampleGenerator.ts

```diff
@@ -106,7 +106,8 @@ export class ExampleGenerator {
                 },
                 snippet: FernGeneratorExec.EndpointSnippet.ruby({
                     client: exampleNode.write({ startingTabSpaces: 0 })
-                })
+                }),
+                exampleIdentifier: endpoint.name.originalName
```

Comment:
we are setting the exampleIdentifier to the original name of the endpoint -- this should be the name of the example. 

I'd also expect to see a for loop somewhere where we iterate over each example, does that already exist?

Reply:
Yeah fixing ruby right now